src/main/java/com/google/sps/data/Destination.java

```diff
@@ -61,6 +61,19 @@ public String getCity() {
     return this.city;
   }
 
+  public String getDescription() {
+    return this.description;
+  }
+
+  public LatLng getLocation() {
+    return this.location;
+  }
+
+  /* Return one Riddle. TODO: Make this a random riddle. */
+  public Riddle getRiddle() {
```

Comment:
I think this would be better as `getRandomRiddle()`, but that change can wait until you actually make it random.